app/utils/jwt_utils.py

```diff
@@ -0,0 +1,37 @@
+from flask_jwt_extended import JWTManager
```

Comment:
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

You should put a blank line between each group of imports.

Read https://www.python.org/dev/peps/pep-0008/#imports